osint/README.md

```diff
@@ -4,6 +4,7 @@ Open-source intelligence (OSINT) is data collected from open source and publicly
 
 ## Passive Recon Tools:
 - [AMass](https://github.com/OWASP/Amass)
+- [Deepinfo](https://deepinfo.com)
```

Comment:
This looks to be a commercial tool. It may be worth mentioning that here, since all others are open source. I can do it after I merge this pull request.

Reply:
Thanks!